pressurecooker/html.py

```diff
@@ -0,0 +1,69 @@
+import os
+import tempfile
+import zipfile
+
+from bs4 import BeautifulSoup
+
+
+class HTMLParser:
+    def __init__(self, filename=None, html=None):
+        self.filename = filename
+        self.html = html
+        self.link_tags = {
```

Comment:
Would it also make sense to add `source`, `iframe`, and `track` tags?

Reply:
Yes, definitely! I left them out for now because I wanted to add some tests with media files, and also to make sure we designed it in such a way that media links could be extracted separately using something like get_media_links. That way, chefs could decide whether to leave the media in the package or to extract it for use elsewhere in Kolibri.

Reply:
Sounds good!